lldb/source/Plugins/Process/Linux/NativeRegisterContextLinuxArm64Shared.h

```diff
@@ -0,0 +1,23 @@
+#include "Plugins/Process/Linux/NativeProcessLinux.h"
+#include "Plugins/Process/Utility/NativeRegisterContextDBReg.h"
+#include "lldb/Utility/Status.h"
+#include <asm/ptrace.h>
+#include <cstdint>
+#include <elf.h>
+#include <sys/ptrace.h>
+#include <sys/uio.h>
+
+namespace lldb_private {
+namespace process_linux {
+namespace arm64 {
+
+Status ReadHardwareDebugInfo(::pid_t tid, uint32_t &max_hwp_supported,
+                             uint32_t &max_hbp_supported);
+
+Status WriteHardwareDebugRegs(
+    int hwbType, ::pid_t tid, uint32_t max_supported,
+    const std::array<NativeRegisterContextDBReg::DREG, 16> &regs);
+
+} // namespace arm64
+} // namespace process_linux
+} // namespace lldb_private
```

Comment:
Add a newline at the end of the file.

I'm of the camp that wonders why this is needed in this day and age, but it does trip up some tools and it's easy to fix.